controllers/flinkcluster_reconciler.go

```diff
@@ -413,7 +413,7 @@ func (reconciler *ClusterReconciler) reconcileJob() (ctrl.Result, error) {
 	var desiredJob = reconciler.desired.Job
 	var observed = reconciler.observed
 	var observedJob = observed.job
-	var recordedJobStatus = observed.cluster.Status.Components.Job
+	var observedJobStatus = observed.cluster.Status.Components.Job
```

Comment:
Why rename it? There are 2 job status: the one recorded in the CR, the one observed from the Flink API.

Reply:
Sorry, it's a mistake. Fixed.